crates/pessimistic-proof/tests/test_utils/forest.rs

```diff
@@ -1,21 +1,43 @@
 use std::collections::{BTreeMap, BTreeSet};
 
+use ethers_signers::{LocalWallet, Signer};
 use pessimistic_proof::{
     bridge_exit::{BridgeExit, LeafType, NetworkId, TokenInfo},
-    imported_bridge_exit::ImportedBridgeExit,
-    keccak::Digest,
+    imported_bridge_exit::{commit_imported_bridge_exits, ImportedBridgeExit},
+    keccak::{keccak256_combine, Digest},
     local_balance_tree::{LocalBalancePath, LocalBalanceTree, LOCAL_BALANCE_TREE_DEPTH},
     local_exit_tree::{data::LocalExitTreeData, hasher::Keccak256Hasher, LocalExitTree},
     multi_batch_header::MultiBatchHeader,
     nullifier_tree::{FromBool, NullifierKey, NullifierPath, NullifierTree, NULLIFIER_TREE_DEPTH},
     utils::smt::Smt,
     LocalNetworkState,
 };
-use rand::random;
-use reth_primitives::U256;
+use rand::{random, thread_rng};
+use reth_primitives::{Address, Signature, U256};
 
 use super::sample_data::{NETWORK_A, NETWORK_B};
 
+pub fn compute_signature_info(
+    new_local_exit_root: Digest,
+    imported_bridge_exits: &[(ImportedBridgeExit, NullifierPath<Keccak256Hasher>)],
+) -> (Digest, Address, Signature) {
+    let imported_hash =
+        commit_imported_bridge_exits(imported_bridge_exits.iter().map(|(exit, _)| exit));
+    let combined_hash =
+        keccak256_combine([new_local_exit_root.as_slice(), imported_hash.as_slice()]);
+
+    let wallet = LocalWallet::new(&mut thread_rng());
```

Comment:
maybe nitpicky, although `thread_rng` implements `CryptoRng` trait, it usually is preferred to expose clearly the `Rng` implementation as part of the function definition so that it can be tweaked at call sites if needed. But non blocking (and I don't know if this affects the rest of the codebase)

Reply:
This is just a test util fn so I can't say this bothers me too much